docs/Contributing/Release.md

```diff
@@ -0,0 +1,143 @@
+# Release Process
+
+
+## Prerequisites:
+- Prepare release notes
+
+
+## Weekly chores
+### Enarx dependency update
+- Update local code
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  ```
+- Checkout chore branch
+  ```bash
+  git checkout -b update-prerequisites
+  ```
+- Check to see `Cargo.toml` is refering to the latest updated prerequisite crates
+- Run cargo-update on all individual sub-crates in updated packages are referenced
+   ```bash
+   for d in internal/*/ ; do (cd "$d" && cargo update); done
+   ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx sub-crate dependencies'
+   ```
+- Run cargo update on `enarx`
+  ```bash
+  cargo update
+  ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx dependencies'
+   ```
+- Run build and tests
+  ```bash
+  cargo clean
+  cargo build
+  cargo tests
+  ```
+- Create PR
+  ```bash
+  git push origin update-prerequisites
+  ```
+
+
+## Enarx Release
+
+### Update and release prerequiste crates (this may be an optional step)
+This includes `xsave`, `sallyport`, etc crates:
+- Ensure all approved PRs are merged
+- Get latest updates and checkout branch
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  git checkout -b update/deps
+  ```
+- Update dependencies
+  ```bash
+  cargo update
+  ```
+- Determine if crate builds and if it works
+  ```bash
+  cargo clean
+  cargo test
+  ```
+- Determine version bump by reviewing output of `git log`
+- Update and bump version in `Cargo.toml`
+- Run `cargo test` again
+- Commit change and push to repo
+  ```bash
+  git commit -asS -m 'Release vX.X.X'
```

Comment:
Maybe we should also have a convention here? E.g.
```suggestion
  git commit -asS -m 'release: vX.X.X'
```

Reply:
I've been following @npmccallum's convention here.